app/src/androidTest/java/com/sdp/movemeet/view/navigation/security/ActivityDescriptionSecurityTest.java

```diff
@@ -15,6 +18,12 @@
 
 public class ActivityDescriptionSecurityTest {
 
+    @Before
+    public void checkUserNotLogged() {
+        FirebaseAuth firebaseAuth = FirebaseAuth.getInstance();
+        if (firebaseAuth.getCurrentUser() != null) firebaseAuth.signOut();
```

Comment:
So here, we assume that user is log by default ? Is it not a strange behavior or assumption ?

Reply:
It's not really that we assumer that user is logged, it's simply checking that the user is not logged so that this test has the right behavior. Sure, we could assume that the user is never logged when lauching a test, but this is not a guarantee. As explained above, I mainly did that to fit the fact that I don't logout the user from the map. But I might change this, in which case we could get rid of this check I guess